<launch>
	<!-- 	dynamic argumanets 192.168.10.31-->
	<!-- <arg name="robot_ip"  value="192.168.10.31"/> -->
	<!-- controller: Controller name (yrc1000) -->
	<!-- <arg name="controller" value="yrc1000"/> -->
	<arg name="robot_type" />

	<arg name="package_path" value="(find motoman_$(arg robot_type)_support)"/>
	<arg name="dollar" value="$"/>

	<!-- robot streaming  -->
    <!-- <rosparam command="load" file="$(arg dollar)$(arg package_path)/config/joint_names_$(arg robot_type).yaml" /> -->

	<!-- <include file="$(find motoman_driver)/launch/robot_state_$(arg controller).launch"> -->
		<!-- <arg name="robot_ip"   value="$(arg robot_ip)" /> -->
	<!-- </include> -->


	<include file="$(arg dollar)$(arg package_path)/launch/load_$(arg robot_type).launch" />
	<!-- 	trajectory simulation -->
<!-- 	<param name="use_gui" value="true" /> -->
	<node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" >
<!-- 		<rosparam param="source_list">["input_value" ,"now_joint_states"]</rosparam> -->
		<rosparam param="source_list">["input_value"]</rosparam>
<!-- 		<remap from="joint_states" to="all_joint_states" /> -->
	</node>

	<node name="robot_state_publisher" pkg="robot_state_publisher" 
		type="robot_state_publisher" required= "true"/>
<!-- 		<remap from="joint_states" to="all_joint_states" /> -->

 
</launch>
